taskcluster/ci/signing/kind.yml

```diff
@@ -20,7 +20,7 @@ job-template:
     description: Sign Fenix
     worker-type:
         by-build-type:
-            (fennec-.+|nightly|beta|production|android-test-nightly):
```

Comment:
should we still be keeping the fennec-beta signing?

Reply:
Good news: `fennec-beta` signing is captured by the `fennec-.+` regex 🙂